src/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+ReactDOM.render(
+  <div>
+
+  </div>,
```

Comment:
This was set up for starting React app's from scratch. I'd say we should either transition it to CRA or leave it a blank slate. Any middle ground doesn't seem beneficial to me with the upcoming curriculum changes.